src/api/index.js

```diff
@@ -379,3 +379,14 @@ export const fetchNotificationsCount = accessToken =>
 
 export const fetchRepoNotificationsCount = (owner, repoName, accessToken) =>
   v3.count(`/repos/${owner}/${repoName}/notifications?per_page=1`, accessToken);
+
+export const fetchIssueEvents = (
+  owner: string,
+  repoName: string,
+  issueNum: number,
+  accessToken: string
```

Comment:
Types make function declaration clearer. But in other functions we never write them.